src/BenchmarkDotNet/Properties/AssemblyInfo.cs

```diff
@@ -15,10 +15,14 @@
 [assembly: InternalsVisibleTo("BenchmarkDotNet.Diagnostics.Windows,PublicKey=" + BenchmarkDotNetInfo.PublicKey)]
 [assembly: InternalsVisibleTo("BenchmarkDotNet.Diagnostics.dotTrace,PublicKey=" + BenchmarkDotNetInfo.PublicKey)]
 [assembly: InternalsVisibleTo("BenchmarkDotNet.IntegrationTests.ManualRunning,PublicKey=" + BenchmarkDotNetInfo.PublicKey)]
+[assembly: InternalsVisibleTo("BenchmarkDotNet.IntegrationTests.ClassicNetFramework,PublicKey=" + BenchmarkDotNetInfo.PublicKey)]
+[assembly: InternalsVisibleTo("BenchmarkDotNet.IntegrationTests.CsProjMultipleFrameworks,PublicKey=" + BenchmarkDotNetInfo.PublicKey)]
```

Comment:
is this test going to be executed by the CI? https://github.com/dotnet/BenchmarkDotNet/blob/6fa7a91444b01029cc2ed81c90e237ae3e36c595/build/BenchmarkDotNet.Build/Runners/UnitTestRunner.cs#L21-L27

Reply:
No, these are all manual tests.